content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -310,6 +310,18 @@ assuming that the number of replicas is not also changed).
 A ReplicaSet can be easily scaled up or down by simply updating the `.spec.replicas` field. The ReplicaSet controller
 ensures that a desired number of Pods with a matching label selector are available and operational.
 
+When scaling down, the ReplicaSet controller chooses which pods to delete by sorting the available pods to
+prioritize scaling down pods based on the following general algorithm:
+ 1. Pending (and unschedulable) pods are scaled down first
+ 2. If controller.kubernetes.io/pod-deletion-cost annotation is set, then
```

Comment:
We should also list `controller.kubernetes.io/pod-deletion-cost` in https://kubernetes.io/docs/reference/labels-annotations-taints/

Reply:
I will add that in https://github.com/kubernetes/website/pull/26739

@alculquicondor we should do that for the indexed job annotation too.

Reply:
Opened #27106